src/enums/Misc.ts

```diff
@@ -0,0 +1,61 @@
+export enum MiscEnum {
```

Comment:
I think you could give better names to your enums.
App.ts, Misc.ts are really not well separated.
add as many enums as you need, there is no need to be cheap with it.